### PR TITLE
Add Placeholder component

### PR DESCRIPTION
**Pull request by stephenjwatkins**
Adds (updates) the Placeholder component to exercise the infrastructure.

- Utilizes tokens
- Utilizes icons (updates icon package export config)
- Updates TypeScript config to use `node16` module resolution
- Add SCSS support
- Adds Vitest
- Update turbo
- Remove unused icons and add used icons
- Update READMEs
- Add Chromatic workflow

### CODE REVIEW

File: .github/workflows/chromatic.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+          script: |
+            await github.rest.repos.createCommitStatus({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              sha: context.sha,
+              state: "success",
+              target_url: "${{ steps.chromatic.outputs.storybookUrl }}",
+              description: "Storybook preview generated!",
+              context: "Chromatic / storybook (push)"
+            });
```
Comment (stephenjwatkins) [inline .github/workflows/chromatic.yml:35]: at some point (once we are sure of using Chromatic) we can properly link Github to Chromatic and do away with the manual status checks

File: easy-ui-icons/README.md [hunk 2]
```diff
@@ -42,13 +42,13 @@ Import the `Icon` component from Easy UI React and any icon from Easy UI Icons i
 2. Import an icon from Easy UI Icons:
 
    ```tsx
-   import ExpandMoreIcon from "@easypost/easy-ui-icons/ExpandMore";
+   import CheckCircleIcon from "@easypost/easy-ui-icons/CheckCircle";
```
Comment (stephenjwatkins) [inline easy-ui-icons/README.md:45]: removed the test icons and added icons that are actually in the Figma

File: easy-ui-react/src/components/Placeholder/Placeholder.module.css [hunk 3]
```diff
@@ -1,3 +0,0 @@
-.Placeholder {
```
Comment (stephenjwatkins) [inline easy-ui-react/src/components/Placeholder/Placeholder.module.css:1]: replaced with SASS module

File: easy-ui-react/src/components/Placeholder/Placeholder.module.scss [hunk 4]
```diff
@@ -0,0 +1,38 @@
+@import "../../styles/common";
```
Comment (stephenjwatkins) [inline easy-ui-react/src/components/Placeholder/Placeholder.module.scss:1]: in this file are styles to test our tokens/css/sass build, they aren't meant to represent all established practices

File: easy-ui-tsconfig/base.json [hunk 5]
```diff
@@ -5,7 +5,7 @@
     "isolatedModules": true,
     "experimentalDecorators": true,
     "lib": ["dom", "es2020"],
-    "moduleResolution": "node",
+    "moduleResolution": "node16",
```
Comment (stephenjwatkins) [inline easy-ui-tsconfig/base.json:8]: need `node16` to be able to interpret the `package.json#exports` field

File: package.json [hunk 6]
```diff
@@ -29,12 +29,13 @@
...
-    "clean": "turbo run clean && rm -rf ./node_modules",
+    "clean": "FORCE_COLOR=1 turbo run clean && rm -rf ./node_modules",
     "format": "prettier . --write",
     "start:storybook": "start-storybook -p 9003",
     "build:storybook": "build-storybook",
-    "build:icons": "turbo run build --filter=@easypost/easy-ui-icons",
-    "build:tokens": "turbo run build --filter=@easypost/easy-ui-tokens"
+    "build:icons": "FORCE_COLOR=1 turbo run build --filter=@easypost/easy-ui-icons",
+    "build:tokens": "FORCE_COLOR=1 turbo run build --filter=@easypost/easy-ui-tokens",
+    "test": "FORCE_COLOR=1 turbo run test"
```
Comment (stephenjwatkins) [inline package.json:38]: need `FORCE_COLOR=1` before turbo commands to preserve colors in the console

File: easy-ui-react/src/components/Placeholder/Placeholder.module.scss (hunk 4 above)
Comment (OskiTheCoder) [reply easy-ui-react/src/components/Placeholder/Placeholder.module.scss:1]: got it, makes sense

File: easy-ui-react/src/components/Placeholder/Placeholder.test.tsx [hunk 7]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { Placeholder } from "./Placeholder";
+
+describe("Placeholder", async () => {
+  it("should render the placeholder message", () => {
+    render(<Placeholder status="success" message={<p>Test message</p>} />);
+    expect(screen.getByText("Test message")).toBeInTheDocument();
+  });
+});
```
Comment (OskiTheCoder) [inline easy-ui-react/src/components/Placeholder/Placeholder.test.tsx:10]: i know this is just a placeholder, but when it comes to tests do you have an idea in mind for how they should be written for the components that will live in easy-ui?

**Review by OskiTheCoder — APPROVED** [review]
nice!

File: easy-ui-react/src/components/Placeholder/Placeholder.test.tsx (hunk 7 above)
Comment (stephenjwatkins) [reply easy-ui-react/src/components/Placeholder/Placeholder.test.tsx:10]: good question. i'm not sure exactly yet on the specifics. probably another good thing to figure out conventions for as we go

**Pull request merged**